tests/libtest.sh

```diff
@@ -86,7 +81,7 @@ cleanup() {
   # cases it has been observed that udev rule kept the device
   # busy.
   udevadm settle
-  clean_config_files $INFILE $OUTFILE
+  rm -f $INFILE $OUTFILE
```

Comment:
ok, these uppercase variable names confused me. Again, can we keep lowercase for local variables and uppercase for gloabl variables.